lib/api/bucketHead.js

```diff
@@ -30,7 +30,8 @@ function bucketHead(authInfo, request, log, callback) {
             authInfo,
             bucket: bucketName,
         });
-        return callback(null, corsHeaders);
+        const headers = { 'x-amz-bucket-region': bucket._locationConstraint };
```

Comment:
logic looks good, however the model provides a getter for this.
https://github.com/scality/Arsenal/blob/6347358cc22c4f2703f0ff309b006f1842bd1e40/lib/models/BucketInfo.js#L535

Reply:
thanks, didn't find it at first